lightning/src/chain/keysinterface.rs

```diff
@@ -379,6 +398,7 @@ impl KeysInterface for KeysManager {
 			delayed_payment_base_key,
 			htlc_base_key,
 			commitment_seed,
+			remote_funding_pubkey,
```

Comment:
style nit: you can just do remote_funding_pubkey: None here instead of initializing further up, but it doesn't matter much.

Reply:
done